### ISSUE TITLE
Stream a specifics user accounts tweets (but not anything else)

### ISSUE THREAD

**Issue by Ev1dentFir3** [desmondmorris/node-twitter#250]
So I am trying to track the tweets a single account. I'm able to stream with
`twitter.stream('statuses/filter', {track: 'WarframeAlerts'},  function(stream)`
but it's also spitting out every tweet other users send them. 

How can I make it so that it only shows the tweets, and not the tweets they receive.
Thanks

**Comment by vallel** [comment]
You should use `follow` (filters by users) instead of `track` (filters by keywords or phrases) in the end point parameters. 
From: https://developer.twitter.com/en/docs/tweets/filter-realtime/api-reference/post-statuses-filter.html

**Comment by dandv** [comment]
Exactly. Here's a [demo](https://github.com/draftbit/twitter-lite/blob/8e5b374ecbfb32d19a2448a2dac3ad0cc34c5871/test/stream.test.js#L28-L34) with **twitter-lite**, which is a modern and actively maintained Twitter library.
